graphics/libpixman-devel/Portfile

```
@@ -35,6 +35,9 @@ long_description        libpixman is a generic library for manipulating pixel \
 
 patchfiles              patch-pixman-pixman-vmx.c.diff
 
+depends_build-append \
+                        port:pkgconfig
```

port:pkgconf also provides `pkg-config` so I've been switching to this:
```suggestion
                        path:bin/pkg-config:pkgconfig
```

---

> port:pkgconf also provides `pkg-config` so I've been switching to this:
> ```suggestion
>                         path:bin/pkg-config:pkgconfig
> ```

Sounds good, I'll use a path dep going forward.